stdlib/public/Darwin/CoreGraphics/CGFloat.swift.gyb

```diff
@@ -510,12 +510,12 @@ public func %=(lhs: inout CGFloat, rhs: CGFloat) {
 }
 
 //===----------------------------------------------------------------------===//
-// Real conformance
+// RealProtocol conformance
```

Comment:
When I looked at this diff I thought the `Real` protocol got renamed to `RealProtocol`.

Reply:
Fixed.